backend-node/helpers/jwt.js

```diff
@@ -0,0 +1,34 @@
+import { tokenGenerate } from '@vonage/jwt';
+
+const appId = process.env.VCR_API_APPLICATION_ID || process.env.appId;
+const privateKey = process.env.VCR_PRIVATE_KEY || process.env.privateKey;
+
+const aclPaths = {
+  "paths": {
+    "/*/users/**": {},
+    "/*/conversations/**": {},
+    "/*/sessions/**": {},
+    "/*/devices/**": {},
+    "/*/image/**": {},
+    "/*/media/**": {},
+    "/*/applications/**": {},
+    "/*/push/**": {},
+    "/*/knocking/**": {},
+    "/*/legs/**": {}
+  }
+};
+
+const getAdminJWT = () => {
+  return tokenGenerate(appId, privateKey);
```

Comment:
Think we should have a ttl explicitly stated for the admin JWT?

Reply:
It seems that without it, it lasts for 900 seconds which is fine I think. https://github.com/Vonage/vonage-node-sdk/blob/153fc39c21f9bc1b12ed3cbd872b3e68bc572328/packages/jwt/lib/tokenGenerate.ts#L26C28-L26C31